Cargo.toml

```diff
@@ -1,8 +1,8 @@
 
 [package]
 name = "mavlink"
-version = "0.7.0"
-authors = ["Todd Stellanova", "Michal Podhradsky", "Kevin Mehall", "Tim Ryan", "Patrick José Pereira"]
+version = "0.8.0"
```

Comment:
@patrickelectric should this be a major or minor release?

Reply:
It depends if it'll break or change the API, from what I can see, the common prefix will be any dialect or all dialects that the feature enable. I believe that the correct way should be a prefix for eat dialect and enable it if such feature is selected.